action.yml

```diff
@@ -119,7 +119,7 @@ runs:
           cd /tmp
           curl -sfL https://raw.githubusercontent.com/securego/gosec/master/install.sh | sh -s v2.13.1
           chmod +x ./bin/gosec
-          curl -fsSL https://api.github.com/repos/codacy/codacy-gosec/releases/latest \
+          curl -fsSL https://api.github.com/repos/codacy/codacy-gosec/releases/latest -H "Authorization: Bearer  ${{ secrets.GITHUB_TOKEN }}" \
```

Comment:
We use this `action` and is now broken. I think is related to this PR.

I am afraid that you cannot access `secrets` from this action this way due to security reasons (if you could, you could basically read all the secrets you wanted). As per the [docs](https://docs.github.com/en/actions/using-workflows/reusing-workflows#using-inputs-and-secrets-in-a-reusable-workflow) you can (1) either ask the users to use the new option of `secrets: inherit` (which might have [some issues btw](https://github.com/orgs/community/discussions/23107)), or (2) ask the user to pass the token as an input -which is the common practice- here.

Reply:
Thanks for bringing this to attention. The change is being reverted.